perl_mongers.xml

```diff
@@ -16,10 +16,15 @@
       <latitude>40.72426868</latitude>
     </location>
     <web>https://groups.google.com/g/new-york-perlmongers</web>
+    <tsar>
+      <name>James E Keenan</name>
+      <email type="personal">
+        <user>jkeenan</user>
+        <domain>pobox.com</domain>
+    </tsar>
```

Comment:
```suggestion
      </email>
    </tsar>
```

Reply:
If you click the "Commit suggestion" button I think you will properly close the `<email>` tag?